data/json/items/gun/45.json

```diff
@@ -84,15 +86,17 @@
     "type": "GUN",
     "name": "L39B-45",
     "description": "After the success of the 9mm model, Leadworks LLC released an L39B chambered in .45.  It soon earned a reputation as one of the most intimidating guns on the market, for firer and target alike.",
-    "ammo": "45"
+    "ammo": "45",
+    "ranged_damage": 1
   },
   {
     "id": "l_sp_45",
     "copy-from": "l_sp_9mm",
     "type": "GUN",
     "name": "L39-45",
     "description": "Capitalizing on the L39B-45's reputation as one of the most intimidating pistols on the market, Leadworks LLC manufactured a semiautomatic model as well.  Locking it semi-auto reduced cost and improved performance.",
-    "ammo": "45"
+    "ammo": "45",
+    "ranged_damage": 1
```

Comment:
That's actually the result of a recent change on your part: The L39-45 and L39-45B used to be Volume 3 weapons (and still were at the time I started working on this). As volume 2, yeah, 0 damage fits better.